src/openrct2/Context.cpp

```diff
@@ -148,13 +148,14 @@ namespace OpenRCT2
             // NOTE: We must shutdown all systems here before Instance is set back to null.
             //       If objects use GetContext() in their destructor things won't go well.
 
+            network_close();
```

Comment:
Perhaps add a comment to explain why it needs to be done before unloading objects, to avoid regressions in the future.